docs/README.md

```diff
@@ -98,17 +98,17 @@ environment:
     github_app_id: "<app_id>"
 ```
 
-* Create a secret for the HMAC / webhook value:
+#### Create a secret for the HMAC / webhook value:
 
-Kubernetes:
+##### Kubernetes:
 
 ```bash
 WEBHOOK_SECRET="Long-Password-Phrase-Goes-Here"
 
 kubectl create secret generic -n openfaas-fn github-webhook-secret --from-literal github-webhook-secret="$WEBHOOK_SECRET"
 ```
 
-Swarm:
+##### Swarm:
 
 ```bash
 WEBHOOK_SECRET="Long-Password-Phrase-Goes-Here"
```

Comment:
we don't mention any but presumably this secret also needs to be updated in the Github App  `Webhook secret (optional)` field?